examples/react-hooks-example/src/post.md

```diff
@@ -0,0 +1,221 @@
+
+### Observable declaration
+
+The below hook defines our 2 observables.
+```js
+    useEffect(() => {
+        web3.eth.getBlockNumber().then((block) => setBlock(block));
+        if (typeof(latestBlock) != "number") 
+            return;
+
+        const EthPurchased$ = daiContract.events.EthPurchase.track({
+            fromBlock: latestBlock - 50
+        });
+        const last5$ = EthPurchased$.pipe($latest(5));
+        setObservable(EthPurchased$);
+        setLast5Observer(last5$)
+    }, [setObservable, setLast5Observer, latestBlock])
+```
+
+`EthPuruchase$` is using the Subspace [trackEvent](https://subspace.embarklabs.io/api.html#contract-methods) method to stream the `EthPurchase` event from the Uniswap exchange contract.  
+
+Important note: When using the `trackEvent` method, make sure to specify the `fromBlock` parameter as I've done in the above code block.  If you don't, it will begin at the very first block where the contract was deployed and stream every event from then on.  In the example here, that could be tens of thousands of events.  The first time I did this, the app basically got stuck endlessly streaming old `EthPurchase` events going back to the beginning of Uniswap.
+
+`last5$` creates a new observable that returns the last 5 trades streamed from `EthPurchase$` using the [`latest` operator](https://github.com/embarklabs/subspace/blob/353fee7d23f12ffa3a8f91fe095500335016586e/packages/core/src/operators.js#L84) that is exposed by Subspace.
+
+### Setting up subscriptions
+
+Now, let's fire up the subscriptions.
+
+First, let's start streaming `EthPurchase` events.  This is a very simple example and just starts up the subscription and prints each transaction to the developer console.  An important thing to remember is to pass the observable's `unsubscribe` function in the return value of the effect.  This is an important feature of Effect hooks.  The hook runs at each component re-render or whenever an element in the dependency array changes and so its important to clean up the subscription before the re-render to avoid memory leaks.  React will call the function specified in the return before re-running the hook during re-render.
+
+```js
+    useEffect(() => {
+        if ((txnObserver === undefined) || (typeof latestBlock != "number")) {
+            return;
+        } else {
+            txnObserver.subscribe((trade) => {
+                console.log(trade);
+            });
+        }
+        return txnObserver.unsubscribe;
+    }, [txnObserver, latestBlock]);
+```
+
+Next, the magic.  Our `last5Observer` observable is the one that produces the data we actually use in the dapp's UI.
+
+```js
+    useEffect(() => {
+        if (last5Observer === undefined) {
+            return;
+        } else {
```

Comment:
No `else` needed, just the early return.

Reply:
Fixed in next commit